### PR TITLE
docs(next/readme): add build instructions for the Markdown backend

### PR DESCRIPTION
**Pull request by rami3l**

### CODE REVIEW

File: next/README.md [hunk 1]
```diff
@@ -45,9 +45,16 @@ PATH=$PATH:/usr/local/texlive/2024/bin/universal-darwin/ make latexpdf
 open ./_build/latex/moonbitdocument.pdf
 ```
 
+For Markdown:
+
+```bash
+pip install sphinx-markdown-builder
```
Comment (peter-jerry-ye) [inline next/README.md:51]: put it in the requirements.txt?

File: next/README.md [hunk 2]
```diff
@@ -45,9 +45,16 @@ PATH=$PATH:/usr/local/texlive/2024/bin/universal-darwin/ make latexpdf
...
+sphinx-build -M markdown . ./_build
```
Comment (peter-jerry-ye) [inline next/README.md:52]: does `make markdown` work?

File: next/README.md (hunk 1 above)
Comment (rami3l) [reply next/README.md:51]: Done!

File: next/README.md (hunk 2 above)
Comment (rami3l) [reply next/README.md]: It does work. Replaced.

**Review by peter-jerry-ye — APPROVED** [review]

**Pull request merged**